docs/developer/base/logs-crawlers.md

```diff
@@ -0,0 +1,47 @@
+# Log Crawlers
+
+## Overview
+
+Some systems expose their logs from HTTP endpoints instead of files that the Logs Agent can tail.
+In such cases you can create an agent integration to crawl the endpoints and submit the logs.
+
+This diagram shows how a logs crawler agent integration that fits into the agent.
+
+<div align="center" markdown="1">
+
+```mermaid
+graph LR
+    subgraph "Agent Integration (you write this)"
+    A[Log Stream] -->|Log Records| B(Log Crawler Check)
+    end
+    subgraph Agent
+    B -->|Send Logs| C(RTLoader API)
```

Comment:
Let's not mention the API (especially because the name will change after a new implementation) and just indicate that it writes directly to a log file

Reply:
👍